app/javascript/elm/src/MobileSessionsFilters.elm

```diff
@@ -142,6 +195,68 @@ view model =
         ]
 
 
+viewParameterFilter : String -> Html Msg
+viewParameterFilter selectedParameter =
+    div []
+        [ h4 [] [ text "parameter" ]
+        , input
+            [ Attr.id "parameter-filter"
+            , clickWithoutDefault ShowSelectFormItemsPopup
+            , Attr.value selectedParameter
+            ]
+            []
+        ]
+
+
+viewPopup : (String -> Msg) -> Bool -> List String -> Html Msg
+viewPopup onSelect isPopupExtended items =
+    let
+        numberOfMainItems =
+            4
+
+        mainItems =
+            List.take numberOfMainItems items
+
+        moreItems =
+            List.drop numberOfMainItems items
```

Comment:
is there an easier way to split a list into two parts on a given index?

Reply:
Would this work? https://package.elm-lang.org/packages/elm-community/list-extra/latest/List-Extra#splitAt